Support for GHC-8.10.2

GHC-8.10.2 was just released. 

Hopefully, the differences between it and GHC-8.10.1 from the HLS point of view are minor enough to make adding the support easy?

To illustrate better (`master` branch, haven't gotten around to `hls-3` yet):
```
$ ./cabal-hls-install ghcs
Resolving dependencies...
Build profile: -w ghc-8.10.1 -O1
In order, the following will be built (use -v for more details):
 - fake-package-0 (exe:script) (configuration changed)
Configuring executable 'script' for fake-package-0..
Preprocessing executable 'script' for fake-package-0..
Building executable 'script' for fake-package-0..
[1 of 1] Compiling Main             ( Main.hs, /Users/ur20980/src/haskell-language-server/dist-newstyle/build/x86_64-osx/ghc-8.10.1/fake-package-0/x/script/build/script/script-tmp/Main.dyn_o )

Main.hs:20:1: warning: [-Wmissing-signatures]
    Top-level binding with no type signature: main :: IO ()
   |
20 | main = defaultMain
   | ^^^^
Linking /Users/ur20980/src/haskell-language-server/dist-newstyle/build/x86_64-osx/ghc-8.10.1/fake-package-0/x/script/build/script/script ...

********************************************************************************
Found the following GHC paths: 
ghc-8.10.1: /Users/ur20980/.ghcup/bin/ghc-8.10.1
ghc-8.6.5: /Users/ur20980/.ghcup/bin/ghc-8.6.5
ghc-8.8.3: /Users/ur20980/.ghcup/bin/ghc-8.8.3
ghc-8.8.4: /Users/ur20980/.ghcup/bin/ghc-8.8.4

********************************************************************************

Build completed in 0.00s

$ type ghc-8.10.2
ghc-8.10.2 is /Users/ur20980/.ghcup/bin/ghc-8.10.2
$ type ghc-8.10.1
ghc-8.10.1 is /Users/ur20980/.ghcup/bin/ghc-8.10.1
$ 
```

--- a/stack-8.10.1.yaml
+++ b/stack-8.10.1.yaml
@@ -1,4 +1,4 @@
-resolver: nightly-2020-06-29
+resolver: nightly-2020-08-08
 
 packages:
 - .
@@ -12,24 +12,16 @@ extra-deps:
 - github: bubba/brittany
   commit: c59655f10d5ad295c2481537fc8abf0a297d9d1c
 - Cabal-3.0.2.0
-- hie-bios-0.6.1
-- cabal-plan-0.7.0.0
 - clock-0.7.2
 - data-tree-print-0.1.0.2
 - floskell-0.10.4
 - fourmolu-0.1.0.0@rev:1
-- ghc-exactprint-0.6.3
 - HsYAML-aeson-0.2.0.0@rev:2
-- lens-4.19.1
-- lsp-test-0.11.0.4
 - monad-dijkstra-0.1.1.2
-- optics-core-0.3
-- ormolu-0.1.2.0
 - retrie-0.1.1.1
 - stylish-haskell-0.11.0.3
 - semigroups-0.18.5
 - temporary-1.2.1.1
-- these-1.1
 
 flags:
   haskell-language-server:

--- a/stack-8.10.2.yaml
+++ b/stack-8.10.2.yaml
@@ -1,0 +1,39 @@
+resolver: nightly-2020-08-08
+compiler: ghc-8.10.2
+
+packages:
+- .
+- ./ghcide/
+
+ghc-options:
+  "$everything": -haddock
+
+extra-deps:
+- aeson-1.5.2.0
+- github: bubba/brittany
+  commit: c59655f10d5ad295c2481537fc8abf0a297d9d1c
+- Cabal-3.0.2.0
+- clock-0.7.2
+- data-tree-print-0.1.0.2
+- floskell-0.10.4
+- fourmolu-0.1.0.0@rev:1
+- HsYAML-aeson-0.2.0.0@rev:2
+- monad-dijkstra-0.1.1.2
+- retrie-0.1.1.1
+- stylish-haskell-0.11.0.3
+- semigroups-0.18.5
+- temporary-1.2.1.1
+
+flags:
+  haskell-language-server:
+    pedantic: true
+  retrie:
+    BuildExecutable: false
+
+# for data-tree-print's bounds on base (>=4.8 && <4.14); using base-4.14.0.0.
+allow-newer: true
+
+nix:
+  packages: [ icu libcxx zlib ]
+
+concurrent-tests: false

